torchgeo/datasets/agb_live_woody_density.py

```diff
@@ -83,10 +83,11 @@ def __init__(
         """
         self.root = root
         self.download = download
+        bands: Sequence[str] = []
```

Comment:
Should these be `[]` or `None`? To me, `[]` implies no bands, whereas `None` implies all bands. In fact, we could just not pass `bands` to `__init__` at all. You would need to change `transforms, cache` to `transform=transforms, cache=cache` since the inputs are now out of order. This also raises the question of what we should do when a user uses `bands=[]`. Should we raise an error saying that `bands` can't be empty? What do you think?

Reply:
Since a `RasterDataset` is guaranteed to have at least one band I don't think there is a distinction between `[]` and `None`. If `bands` evaluates to False (`None` or `[]`) we use `all_bands` else we use `bands`.

> In fact, we could just not pass bands to `__init__` at all. You would need to change transforms, cache to `transform=transforms`, `cache=cache` since the inputs are now out of order

Agreed. I'll push the change.